### DISCUSSION TITLE
Issue in importlib_resources encountered when using Hydra, input needed

### DISCUSSION THREAD

**Discussion by MattiasDC** [facebookresearch/hydra#2279] [Q&A]
Hi,

When using hydra with a custom search path plugin, I stumbled upon a regression between `importlib_resources` 5.7 and 5.8. This issue was logged and is being fixed. There are however some questions about possible improvements in Hydra/my plugin, which I cannot confidently answer. Can someone with more knowledge on hydra give some input [here](https://github.com/python/importlib_resources/issues/253)?

**Comment by jaraco** [comment]
My recommendation would be for hydra to specify the `pkg:` scheme as:  `pkg://{package}/{path}` where `{package}` is a dot-separate Python package (which may or may not include a namespace package) and `{path}` which is a `/`-separated resource path within that package.

Then, that scheme would translate directly to an invocation of those resources as `importlib.resources.files(package).joinpath(path)`.
